less/mixins.less

```diff
@@ -1 +1,83 @@
 // Mixins below variables
+
+.width-start-mixin() {
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+ 
+}
+
+.width-start-mixin-footer() {
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+
+.flex-column() {
+    display: flex;
+    flex-direction: column;
+}
+
+.h2-padding() {
+    font-size: @h2-font-size;
+    margin: 20px 0 10px 0;
+}
+
+.p-padding() {
+    font-size: @p-font-size;
+    margin: 0 0 20px 0;
+}
+
+.text-content-width-padding() {
+    width: 75%;
+    padding: 0 5% 0 2%;
+}
```

Comment:
This one would be a good one too. As long as you are using it a few times in a few different places.